xla/service/gpu/fusions/dynamic_slice_fusion_test.cc

```diff
@@ -3269,6 +3269,91 @@ TEST_F(DynamicSliceFusionTest,
       /*run_hlo_passes=*/false, /*use_threads=*/true, error));
 }
 
+TEST_F(DynamicSliceFusionTest, AsyncCollective) {
```

Comment:
Let's give a name that explains the behaviour that we're testing to this test.

Is it that "DynamicSliceFusionIsWrappedInsideAsyncCollective"?